webapp/handlers.py

```diff
@@ -49,20 +48,6 @@
 
 from datetime import datetime
 
-badge_counter = prometheus_client.Counter(
```

Comment:
Are we completely losing those metrics, or will they now be automatically part of 12 factor app o11y?

Reply:
@bartaz I believe this is something we'll be able to get with COS Lite. This doesn't work once talisker is removed.